config.json

```diff
@@ -99,8 +99,11 @@
     }
   },
   "tipsel": {
-    "belowMaxDepthTransactionLimit": 20000,
-    "maxDepth": 5
```

Comment:
why rename maxDepth to belowMaxDepth?

Reply:
maxDepth was another thing back then... it was used for maximum starting depth of the random walk.
This belowMaxDepth is the treshold for lazy tips.